project.clj

```diff
@@ -0,0 +1,46 @@
+;; PROJECT_VERSION is set by .circleci/deploy/deploy_release.clj,
+;; whenever we perform a deployment.
+(defproject haystack (or (not-empty (System/getenv "PROJECT_VERSION"))
```

Comment:
This should be `mx.cider/haystack`. We can no longer push new projects to Clojars without an approved FQ group-id. At some port I'll likely change the older group ids for consistency as well.